goal_orientation:
  w: 0.7073426465368579
  z: 0.7068708371337962
goal_position:
  x: -0.5661862483834903
  y: 1.4733191107227963
